lua/lualine/extensions/fugitive.lua

```diff
@@ -7,8 +7,16 @@ local function fugitive_branch()
   return icon .. ' ' .. vim.fn.FugitiveHead()
 end
 
+local function get_git_toplevel_basename()
+  local handle = io.popen("basename `git rev-parse --show-toplevel`")
```

Comment:
I don't think this works cross-platform.
See if fugitive provides repo name somewhere you can fetch it from there easily in that case.